src/include/execution/vm/bytecode_handlers.h

```diff
@@ -67,6 +67,10 @@ ALL_TYPES(COMPARISONS);
 
 VM_OP_HOT void OpNot(bool *const result, const bool input) { *result = !input; }
 
+VM_OP_HOT void OpNotSql(terrier::execution::sql::BoolVal *result, terrier::execution::sql::BoolVal *input) {
```

Comment:
Make input a `const *` to convey intention.

Reply:
I made result a const * as well. I think when you use * after the type, then it only requires you preserve the pointer address. And this is consistent with the other functions.

Reply:
Change it to `const terrier::execution::sql::BoolVal *input` since it isn't modified.